msu/systems/tooltips/tooltips/basic_tooltip.nut

```diff
@@ -0,0 +1,51 @@
+::MSU.Class.BasicTooltip <- class extends ::MSU.Class.Tooltip
+{
+	Title = null;
+	Text = null;
+
+	constructor(_title, _text, _data = null)
```

Comment:
space before and after args. I will stop posting this comment further. Please go through the entire PR and fix all instances of this.